packages/cli/src/deploy/ensureDeployer.ts

```diff
@@ -1,16 +1,15 @@
-import { Account, Address, Chain, Client, Transport } from "viem";
+import { Account, Address, Chain, Client, Transport, sliceHex } from "viem";
 import { getBalance, getBytecode, sendRawTransaction, sendTransaction, waitForTransactionReceipt } from "viem/actions";
 import deployment from "./create2/deployment.json";
 import { debug } from "./debug";
 
 const deployer = `0x${deployment.address}` as const;
-const deployerBytecode = `0x${deployment.bytecode}` as const;
 
 export async function ensureDeployer(client: Client<Transport, Chain | undefined, Account>): Promise<Address> {
   const bytecode = await getBytecode(client, { address: deployer });
   if (bytecode) {
     debug("found CREATE2 deployer at", deployer);
-    if (bytecode !== deployerBytecode) {
+    if (bytecode !== sliceHex(`0x${deployment.creationCode}`, 14)) {
```

Comment:
what's the 14?

Reply:
<img width="708" alt="image" src="https://github.com/latticexyz/mud/assets/508855/bd464705-57d7-49c8-ad7d-10137eccf5a4">